types/block.go

```diff
@@ -20,7 +20,7 @@ import (
 
 const (
 	// MaxHeaderBytes is a maximum header size (including amino overhead).
-	MaxHeaderBytes int64 = 632
+	MaxHeaderBytes int64 = 628
```

Comment:
Also curious, what's this change about?

Reply:
memory layout in go for int64 is 8 bytes, by reducing it to int32 it becomes 4 bytes. just learned this 😄